DOCS-873 - Remove duplicate information from Cloud SIEM ingestion articles (#5376)
SumoLogic/sumologic-documentation: docs/cse/ingestion/ingestion-sources-for-cloud-siem/aws-cloudtrail.md

---
id: aws-cloudtrail
title: Ingest AWS CloudTrail Data into Cloud SIEM
sidebar_label: AWS CloudTrail

> description: Configure a CloudTrail source on a source to ingest CloudTrail log messages to be parsed by Cloud SIEM's CloudTrail system parser.

Reviewer: ```suggestion
description: Configure a CloudTrail source to ingest log messages that are parsed by Cloud SIEM’s CloudTrail system parser.
```
